src/Ztm.Data.Entity/Contexts/MainDatabase.cs

```diff
@@ -136,12 +136,12 @@ protected virtual void ConfigureWebApiCallbackHistory(ModelBuilder modelBuilder)
         {
             modelBuilder.Entity<WebApiCallbackHistory>(b =>
             {
-                b.Property(e => e.CallbackId).IsRequired().ValueGeneratedNever();
+                b.HasKey(e => e.Id);
```

Comment:
This one should be `b.Property(e => e.Id).IsRequired().ValueGeneratedOnAdd()`.

Reply:
Note: intentionally add `ValueGeneratedOnAdd` explicit to improve readability.